_posts/0000-01-02-himansh107.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: “Welcome to our second slide!”
+---
+Your test
```

Comment:
currently studying in NIT Jsr
![nit](https://user-images.githubusercontent.com/75253218/100730565-389e5400-33f0-11eb-8f98-5b661a34092d.png)

Reply:
seems like a good addition